src/wrapped_radix_engine_toolkit/high_level/sargon_specific_types/assets_transfers/per_asset/per_asset_fungible_transfer.rs

```diff
@@ -0,0 +1,123 @@
+use crate::prelude::*;
+
+impl PerAssetFungibleTransfer {
```

Comment:
The `PerAssetFungibleTransfer ` type is declared with macro in `transfer_types.rs`